docs/manage/data/dataset.md

```diff
@@ -67,6 +73,51 @@ Repeat this with all images in your dataset.
 
 To delete a tag, click the cross next to the corresponding label.
 
+## Datasets
+
+{{< alert title="Support" color="info" >}}
+Datasets currently only work for images.
+{{< /alert >}}
+
+### Create a dataset
+
+To create a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **DATASETS** subtab, click **Create dataset**, enter a name for your dataset and click **Create dataset** again.
```

Comment:
We can add it in explicitly when https://viam.atlassian.net/browse/DATA-2166 is resolved, but might be worth mentioning that it can be created directly on the image, the same way that you add data to a dataset below.

Though I know it's already written below that `You can select an existing dataset or create a new one from this dropdown` so perhaps too redundant?